resources/shaders/dxr_pathtracer_main.hlsl

```diff
@@ -79,10 +80,19 @@ void RaygenEntry()
 
 	float3 result = float3(0, 0, 0);
 
+	SurfaceHit sfhit;
+	sfhit.pos = wpos;
+	sfhit.normal = normal;
+	sfhit.dist = length(cpos - wpos);
+	sfhit.surface_spread_angle = ComputeSurfaceSpreadAngle(gbuffer_depth, gbuffer_normal, inv_vp, wpos, normal);
+
+	// Compute the initial ray cone from the gbuffers.
+ 	RayCone cone = ComputeRayConeFromGBuffer(sfhit, 1.39626, DispatchRaysDimensions().y);
```

Comment:
I don't like the magic number.

Reply:
is already an issue and out of the scope of this pr.

Reply:
Okay, no need to change it, then! :)